data/autoyast_sle15/create_hdd/create_hdd_multipath_lvm_encrypt_textmode_aarch64.xml

```diff
@@ -311,7 +311,6 @@
     <patterns t="list">
       <pattern>apparmor</pattern>
       <pattern>base</pattern>
-      <pattern>basic_desktop</pattern>
```

Comment:
We may need to keep this line for textmode as well :)

Reply:
Added back `basic_desktop`